backend/models/membership.js

```diff
@@ -14,13 +14,17 @@ const membershipSchema = new mongoose.Schema({
         index: true,
         isRequired: true
     },
+    firstName: {
```

Comment:
Considering that `membership` does not take in the `firstName`, would it be ok if I removed this @Donnyd88  and then push another commit?

Reply:
Yes. I should have removed that. Ty.